### PR TITLE
Add katakana charset

### PR DESCRIPTION
**Pull request by adiee5**
This adds a katakana character set, which is based on [JIS X 0201](https://en.wikipedia.org/wiki/JIS_X_0201) standard. Since the standard leaves $E0-$FF empty, this implementation adds miscellaneous kanji and other characters to this space. Those additions are loosely based on additions made in Sharp X1's implementation ($F1-$FF and $E3-$E5 are identical, rest is different)

image

### CODE REVIEW

File: kernal/drivers/x16/screen.s [hunk 1]
```diff
@@ -808,6 +816,58 @@ cpylaeB:	lda #$d8
...
+	ldx #3
+	jsr copyv
+
+	; copying individual characters
+
+@vram_backslash	=$f000+($5c*8)
+@chariso2_yen	=$d800+($a5*8)
+@vram_tilde 	=$f000+($7e*8)
+@charkat_tilde	=$f400+($00*8)
+@vram_nbsp  	=$f000+($a0*8)
```
Comment (mooinglemur) [inline kernal/drivers/x16/screen.s:835]: For the ROM locations of the glyphs, I would rather avoid using constants like this.  cc65 has a nice way of specifying the offset from the beginning of a segment.

Inside the charset-x16.cfgtpl, in the SEGMENTS section, add `define = yes` to each of the segment definitions, and you can then import the symbol for them in this file and define them here.

For instance
`.import __CHARISO2_LOAD__`

`@chariso2_yen = __CHARISO2_LOAD__+(a5*8)`

For the VRAM addresses, we are already doing `.include "io.inc"` in this file, so you can use the constant `charset_addr`

For instance,
`@vram_nbsp = charset_addr+(a0*8)`

Comment (adiee5) [reply kernal/drivers/x16/screen.s:835]: I like that!

Comment (adiee5) [reply kernal/drivers/x16/screen.s:835]: I'll deal with that tomorrow

Comment (adiee5) [reply kernal/drivers/x16/screen.s]: @mooinglemur compiler has trouble figuring out the segment stuff

**Review by mooinglemur — APPROVED** [review]
Thanks!

**Pull request merged**